app/templates/courses/teacher-courses-view.jade

```diff
@@ -50,6 +50,9 @@ block content
                         | Python
                       option(value="javascript")
                         | JavaScript
+                      if me.enableCpp() && (course.id === view.utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE || course.id === view.utils.courseIDs.COMPUTER_SCIENCE_2 || course.id === view.utils.courseIDs.COMPUTER_SCIENCE_3)
```

Comment:
Nitpick: Could this check be lifted into the coffeescript file to make the template easier to read?
Then it can be reused below.

Reply:
you mean run functions in coffeescript and assigned to a variable used in template ?

Reply:
Yes please.